src/utils/MultipartFormDataParser.ts

```diff
@@ -3,10 +3,20 @@ import util from 'util';
 
 import { getEnvOrThrow } from 'app/utils/EnvironmentUtils';
 
+const supportedFileTypes: string[] = getEnvOrThrow('SUPPORTED_MIME_TYPES').split(',');
+
 export const parseFormData = util.promisify(
     multer({
         limits: {
-            fileSize: parseInt(getEnvOrThrow('MAX_FILE_SIZE_BYTES'), 10)
+            fileSize: Number(getEnvOrThrow('MAX_FILE_SIZE_BYTES'))
```

Comment:
What is that change for? Why was that change needed?

Reply:
I was playing around with it and I probably left it like that. But in our case both will return the same, except parseInt takes an extra parameter while Number doesnt

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt

parseInt apparently can have some unpredictable behaviour, for example If parseInt encounters a character that is not a numeral in the specified radix, it ignores it and all succeeding characters and returns the integer value parsed up to that point. If you want the string to fail with NaN if it has other characters in it, Number() may actually be a better choice.